mlflow/onnx/__init__.py

```diff
@@ -519,4 +523,5 @@ def log_model(
         onnx_execution_providers=onnx_execution_providers,
         onnx_session_options=onnx_session_options,
         metadata=metadata,
+        save_as_external_data=save_as_external_data
```

Comment:
it looks like the `**kwargs` get passed through to the `save_model` function here:
https://github.com/mlflow/mlflow/blob/master/mlflow/models/model.py#L600

Reply:
yep! 100%